test/modules/bradc/printModStuff/foo.good

```diff
@@ -35,6 +34,7 @@ Parsing module files:
   $CHPL_HOME/modules/standard/IO.chpl
   $CHPL_HOME/modules/packages/RangeChunk.chpl
   $CHPL_HOME/modules/packages/Search.chpl
+  $CHPL_HOME/modules/standard/Reflection.chpl
```

Comment:
What caused this order to change?  Nothing that I saw in the code I looked through earlier seemed like it should impact module output order